modules/apps/asset/asset-publisher-web/src/main/java/com/liferay/asset/publisher/web/internal/portlet/AssetPublisherPortlet.java

```diff
@@ -319,10 +161,10 @@ public void serveResource(
 			resourceRequest.getResourceID());
 
 		if (resourceID.equals("getFieldValue")) {
-			getFieldValue(resourceRequest, resourceResponse);
+			_getFieldValue(resourceRequest, resourceResponse);
```

Comment:
Hey @regiCesar21, break down this commit into two ones. The first with the rename and the second one with the moving part